src/backend/vulkan/src/lib.rs

```diff
@@ -50,7 +50,7 @@ mod info;
 mod native;
 mod pool;
 mod result;
-mod window;
+pub mod window;
```

Comment:
if `RawSurface` is the only type that needs to be visible, let's re-export it instead of making `window` module public

Reply:
RawSurface is the only one that *doesn't* need to be visible